src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -39,6 +43,42 @@ message StopContainerRequest {
   string container_name = 3;
 }
 
+message GetHostsRequest {
+  string filter = 1;
+
+  int32 age_in_minutes = 2;
+
+  string status = 3;
```

Comment:
what are some examples of status?  wondering where all those values are.

Reply:
in aws, it's spot and scheduled.

Reply:
in aws, it's running, terminated, etc.